### ISSUE TITLE
when I use adam_offload, the training is not going

### ISSUE THREAD

**Issue by AXy1527** [OpenRLHF/OpenRLHF#1049]
when I use adam_offload, the training is not going
my setting:

ray job submit --address="http://127.0.0.1:8265" \
   -- python3 -m openrlhf.cli.train_ppo_ray \
   --ref_num_nodes 1 \
   --ref_num_gpus_per_node 4 \
   --actor_num_nodes 1 \
   --actor_num_gpus_per_node 4 \
   --vllm_num_engines 2 \
   --vllm_tensor_parallel_size 2 \
   --colocate_all_models \
   --vllm_gpu_memory_utilization 0.6 \
   --num_episodes 3 \
   --pretrain  \
   --remote_rm_url ./examples/scripts/reward_func_1.py \
   --save_path ./outputs/deepscaler-1.5B-GRPO-stage_1 \
   --micro_train_batch_size 2 \
   --train_batch_size 128 \
   --micro_rollout_batch_size 4 \
   --rollout_batch_size 128 \
   --n_samples_per_prompt 16 \
   --max_epochs 1 \
   --prompt_max_len 1024 \
   --max_samples 10000 \
   --generate_max_len 4096 \
   --init_kl_coef 1e-4 \
   --gamma 1.0 \
   --use_kl_loss \
   --kl_estimator k3 \
   --advantage_estimator group_norm \
   --zero_stage 3 \
   --bf16 \
   --actor_learning_rate 1e-6 \
   --prompt_data ../experiment/datasets/DeepScaleR-Preview-Dataset/deepscaler.json \
   --input_key problem \
   --label_key answer \
   --normalize_reward \
   --gradient_checkpointing \
   --adam_offload \
   --packing_samples \
   --vllm_sync_backend nccl \
   --enforce_eager \
   --vllm_enable_sleep \
   --deepspeed_enable_sleep \
   --save_steps 100 \
   --ckpt_path ./outputs/deepscaler-1.5B-GRPO-stage_1 \
   --use_tensorboard ./outputs/deepscaler-1.5B-GRPO-stage_1

**Comment by hijkzzz** [comment]
please remove --deepspeed_enable_sleep
